src/send/SendEnterAmount.test.tsx

```diff
@@ -47,7 +47,7 @@ const mockStore = {
       }, // fee currency, matches mockCeloTokenBalance
       [mockEthTokenId]: {
         tokenId: mockEthTokenId,
-        balance: '0',
+        balance: '0.1',
```

Comment:
So, now we don't want to show tokens without balance in the SendEnterAmount, and there was a test failing because Eth wasn't present, so this was the quick fix.
But, maybe I can put this in 0 again and change the failing test a bit.